ConEmuWinForms/ConEmuSession.cs

```diff
@@ -185,10 +185,11 @@ public void CloseConsoleEmulator()
 		{
 			try
 			{
-				if(!_process.HasExited)
-                    Thread.Sleep(10);
-                if (!_process.HasExited)
+				if (!_process.HasExited)
+				{
+					Thread.Sleep(10);

```

Comment:
Now it looks odd. Why should we wait 10msecs before killing the process. I think that the removed double checking was introduced on purpose, not by mistake.